eo-parser/src/test/java/org/eolang/parser/SyntaxTest.java

```diff
@@ -116,44 +116,6 @@ void parsesSuccessfully(final String code) {
         );
     }
 
-    @Test
-    void failsWithDoubleNewLine() {
```

Comment:
@yegor256 I believe that removing tests is a dangerous approach. It is much safer to keep the tests and modify the assertions. For instance, you can verify that the final output includes errors:
```
/program/errors/error[1]
```

Reply:
@volodya-lombrozo these tests duplicate what we already test in `TyposTest`, that's why I removed them